rapidtide/correlate.py

```diff
@@ -94,6 +84,7 @@ def resdec(f):
 
 
 def disablenumba():
```

Comment:
This function isn't used within this module. The only version of this one that does seem to be used is the one in `util.py`, which is used within `rapidtide` (the workflow).

Reply:
I had a plan for using this to systematically disable numba everywhere for testing, but ok to remove it.  My numba implementation is so old, I have to redo everything anyway, and it's currently shut off by default until I get to it.